src/modules/InviteUser/views/ViewInvitations.vue

```diff
@@ -1,26 +1,63 @@
 <script>
-    import InviteUser from '../components/InviteUser.vue'
-    import InvitationActions from '../components/invitationActions.vue';
-    import {invitationList} from '../services'
-    //used for testing
-    import axios from 'axios'
-    export default{
-    components: [
-        InviteUser,
-        InvitationActions
-    ],
-    data() {
-        return {
-            invitations: []
-        };
+import InviteUser from "../components/InviteUser.vue";
+import InvitationActions from "../components/invitationActions.vue";
+import { invitationList } from "../services";
+//used for testing
+import axios from "axios";
+export default {
+  components: [InviteUser, InvitationActions],
+  data() {
+    return {
+      invitations: [],
+    };
+  },
+  components: { InvitationActions, InviteUser },
+    mounted() {
+    this.getInvitationList()
+    //hello
+  },
+  computed: {
+    styleRole() {
+      return (role) => {
+        if (role.toLowerCase() === "employee") {
+          return "role-employee";
+        }
+        if (role.toLowerCase() === "vendor") {
+          return "role-vendor";
+        }
+      };
     },
-    components: { InvitationActions,InviteUser },
-    async created(){
+    styleStatus() {
+      return (status) => {
+        if (status.toLowerCase() === "pending") {
+          return "status-pending";
+        }
+        if (status.toLowerCase() === "expired") {
+          return "status-expired";
+        }
+      };
+    },
+    styleDotIcon() {
+      return (status) => {
+        if (status.toLowerCase() === "pending") {
```

Comment:
create a variable that will contain a lowercase status, instead of using multiple "status.toLowerCase()"
const statusLowerCase = status.toLowerCase();

Reply:
updated